.github/workflows/development.yml

```diff
@@ -8,13 +8,21 @@ jobs:
       run: go version
     - name: Check terraform version
       run: terraform version
+    - name: Create terraform cli config
```

Comment:
what's for is this?

Reply:
actually to create terraform config that have token inside of it, in terraform cloud, it should have terraform team token to init, plan, and apply the terraform. I refer to this https://www.terraform.io/docs/commands/cli-config.html otherwise we can't operate the terraform